dashboard/test/models/levels/blockly_test.rb

```diff
@@ -1238,18 +1238,55 @@ def create_custom_block(name, pool, block_text, args, category: 'Events')
     assert_equal parsed_xml.at_xpath('//block[@type="studio_ask"]/*[@name="VAR"]').content, localized_variable_str
   end
 
-  test 'keeps tags expanded when localizing start_html' do
+  test 'localizes start_libraries' do
```

Comment:
I am considering breaking this test case into 3 different test cases
- levels with translated i18n_libraries
- levels with partially translated i18n_libraries
- levels without i18n_libraries